python/pylibvw.cc

```diff
@@ -416,6 +416,10 @@ size_t my_get_label_type(VW::workspace* all)
   {
     return lCONTINUOUS;
   }
+  else if (lp->parse_label == MULTILABEL::multilabel.parse_label)
+  {
+    return lMULTILABEL;
+  }
```

Comment:
we also need cb_eval here. It may also be better to use lSIMPLE here to phase out lBINARY

Reply:
We could also potentially move to use he output label type of the top of the reduction stack

Reply:
might be risky but that would be cool

Reply:
Yeah lets not for now -  but we can work towards it later